dc/s2n-quic-dc/src/path/secret/map/cleaner.rs

```diff
@@ -104,6 +104,7 @@ impl Cleaner {
         C: 'static + time::Clock + Send + Sync,
         S: event::Subscriber,
     {
+        let start = Instant::now();
```

Comment:
I think i would prefer to get the time through the `state`'s clock just so we're consistent and don't run into non-determinism issues in sims. I know this is just a measured duration right now but if someone adds a metric snapshot it won't be the same value across runs.

Reply:
Yeah, makes sense. Adjusted.